base/shell/cmd/cmd.c

```diff
@@ -180,6 +180,70 @@ CON_STREAM_MODE OutputStreamMode = UTF8Text; // AnsiText;
 WORD wDefColor = 0;     /* Default color */
 #endif
 
+/* Dynamic tracing */
+#ifdef FEATURE_DYNAMIC_TRACE
```

Comment:
@katahiromz Please move all of this new functionality into a new `trace.c` file in the cmd directory.

Reply:
Done.